libs/reputation/src/validators/githubPersonalStars/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Handler } from "../.."
+
+export type Criteria = {
+    minStars: number
+}
+
+const name = "GITHUB_PERSONAL_STARS"
+
+const criteriaABI = {
+    minStars: "number"
+}
+
+/**
+ * It checks if a user has more then 'minStars' stars in their personal repository.
+ * @param criteria The reputation criteria used to check user's reputation.
+ * @param context Utility functions and other context variables.
+ * @returns True if the user meets the reputation criteria.
+ */
+const validate: Handler = async (criteria: Criteria, { utils }) => {
+    utils.checkCriteria(criteria, criteriaABI)
+
+    let allRepositories = []
+
+    for (let i = 0; allRepositories.length % 100 === 0; i += 1) {
+        // eslint-disable-next-line no-await-in-loop
+        const repositories = await utils.githubAPI(
```

Comment:
How about doing these calls in parallel? and use Promise.all()
Does GH impose any limitations on parallel API usage?